storybook/src/components/Menu/Menu.stories.tsx

```diff
@@ -5,11 +5,11 @@
 
 import {
   BarChartIcon,
-  CogwheelIcon,
   DocumentsIcon,
   FolderIcon,
   PieChartFillIcon,
   PieChartIcon,
+  SettingsIcon,
```

Comment:
I see this was a separate commit; we need to revert it with the rename from Cogwheel.